cirq-core/cirq/experiments/random_quantum_circuit_generation.py

```diff
@@ -596,26 +596,26 @@ def random_rotations_between_grid_interaction_layers_circuit(
     qubits = list(qubits)
     coupled_qubit_pairs = _coupled_qubit_pairs(qubits)
 
-    circuit = circuits.Circuit()
-    previous_single_qubit_layer = circuits.Moment()
+    moments: List['cirq.Moment'] = []
+    previous_single_qubit_layer = cirq.Moment()
```

Comment:
nit: revert to using `circuit.Moment()`; `cirq` is imported in an `if TYPE_CHECKING` block, so not available at runtime here.
```suggestion
    previous_single_qubit_layer = circuits.Moment()
```

Reply:
I didn't realized that thank you!